server/src/infra/migrations/1695660378655-fixGPSNullIslandRedux.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class FixGPSNullIslandRedux1695660378655 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `UPDATE "exif" SET latitude = NULL, longitude = NULL WHERE latitude = 'NaN' AND longitude = 'NaN';`,
```

Comment:
Can you just copy `1685731372040-RemoveInvalidCoordinates.ts`?

Reply:
Yeah. I just didn't see it, because the one I copied is more recent.